client/src/sections/Login/LoginModal/LoginModal.tsx

```diff
@@ -0,0 +1,55 @@
+import {Modal, TextField} from '@shopify/polaris';
+import React, {useState, useCallback} from 'react';
+import {useHistory} from 'react-router-dom';
+
+interface Props {
+  active: boolean;
+  handleChange: () => void;
+}
+
+export default function LoginModal({active, handleChange}: Props) {
+  const history = useHistory();
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const handleLogin = () => {
+    history.push('/home');
+  };
+
+  const handleEmailChange  = useCallback((value) => setEmail(value), []);
+  const handlePasswordLogin = useCallback((value) => setPassword(value), []);
+
+  return (
+    <div className='loginModal'>
```

Comment:
I've een naming classes with dashes instead of camelCase. Everyone has their own preference, but since a majority of our css classes are named with dashes, lets stick with that so we can all be consistent: `login-modal`.